File: 1010.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var l1 = lines.shift().split(' ');
var l2 = lines.shift().split(' ');
var a = Number(l1[1]);var b = Number(l1[2]);var c = Number(l2[1]);var d = Number(l2[2]);
console.log(`VALOR A PAGAR: R$ ${((a*b)+(c*d)).toFixed(2)}`);

File: 1010.cs
using System;
class URI {
    static void Main(string[] args) { 
        String[] n1 = Console.ReadLine().Split(' ');
        String[] n2 = Console.ReadLine().Split(' ');
        float total = (int.Parse(n1[1]) * float.Parse(n1[2])) + (int.Parse(n2[1]) * float.Parse(n2[2]));
        Console.WriteLine($"VALOR A PAGAR: R$ {total:F2}");

    }

}
